Jucer2Reprojucer/main.cpp

```diff
@@ -1940,6 +1948,12 @@ int main(int argc, char* argv[])
                                     if (value == "gnu++14")
                                       return "GNU++14";
 
+                                    if (value == "c++17")
+                                      return "C++17";
+
+                                    if (value == "gnu++17")
+                                      return "GNU++17";
+
```

Comment:
The Xcode exporter configuration setting "C++ Language Standard" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-29de8ceb6da4050657ca1e601ef42774L458-L468. At that time, Projucer didn't know about C++17 at all, so these 6 lines are actually dealing with unexpected input. Please remove them.